test.js

```diff
@@ -1,6 +1,41 @@
+const os = require("os")
 const test = require("ava")
 const minecraftPath = require(".")
 
-test("main", t => {
-	t.is(typeof minecraftPath(), "string")
+test("type of minecraftPath return", t => {
+	const originalPlatform = process.platform
+
+	const platforms = ["win32", "darwin", "linux", "android"]
+
+	t.plan(platforms.length)
```

Comment:
We don't need to use `t.plan`. See https://github.com/avajs/ava/blob/master/docs/recipes/when-to-use-plan.md